pkg/features/features.go

```diff
@@ -84,7 +84,13 @@ const (
 	// The feature gate takes precedence over the `HVPA` feature gate when they are both enabled.
 	// owner: @ialidzhikov
 	// alpha: v1.95.0
-	VPAAndHPAForAPIServer = "VPAAndHPAForAPIServer"
+	VPAAndHPAForAPIServer featuregate.Feature = "VPAAndHPAForAPIServer"
+
+	// WorkerPoolHashWithNodeAgentSecret enables a new calculation method for the worker pool hash used by provider extensions. All
+	// such extensions must be upgraded to this Gardener version before enabling the feature gate.
```

Comment:
`this Gardener version` is not well defined. Please adapt maybe similarly to how it is worded in `feature_gates.md`.

Reply:
The description is now synced with that in `feature_gates.md`.